liquid.py

```diff
@@ -242,16 +266,28 @@ def deploy():
     consul.set_kv('liquid_domain', liquid_domain)
     consul.set_kv('liquid_debug', liquid_debug)
 
-    for _, path in JOBS:
+    for job, path in JOBS:
+        print(f'Starting {job}...')
         runjob(path)
 
+    collections = get_config(None, 'collection:')
+    for name, settings in collections:
+        print(f'Starting collection-{name}...')
```

Comment:
But we have logging. This should be `log.info("Starting collection-%s ...", name)`.